### ISSUE TITLE
Need to find common ancestor of unions for FSArray elements

### ISSUE THREAD

**Issue by mecst76** [Blulab-Utah/ie-viz#16]
For UIMA descriptor file, the classes that have the OR logical operator cannot be represented in FSArray. You must find the common ancestor of all of the operands and use it as the value of FSArray.

**Comment by jianlins** [comment]
What does this class do? Cover OWL to UIMA Type descriptor? Is it possible to not use XXArray? I know several programs have problems with that features. GATE doesn't interpret that type feature properly. So does Leo. Not sure if they have been fixed or not. But using that often increases complexity.

**Comment by mecst76** [comment]
Yes, this class is meant to convert a domain ontology into a UIMA Type descriptor file. I'm not very familiar with UIMA. I was just using an example descriptor file to build my code. I'm not familiar with XXArray and not sure if UIMA supports that type or not.

**Comment by jianlins** [comment]
XX means "any", e.g. FSArray, StringArray, LongArray. I think it is a variable type that used to define the parameters of AE descriptors. There are many others, such as, uima.cas.String, uima.cas.Long. I think it should be easier and simpler to consider any unknown feature of a concept Type will take  uima.cas.String values.
